lib/browser.ts

```diff
@@ -775,8 +775,8 @@ export class ProtractorBrowser extends Webdriver {
         .then(loadMocks, deferred.reject);
 
     let self = this;
-    function loadMocks(angularVersion: number) {
-      if (angularVersion === 1) {
+    function loadMocks(angularVersion: string) {
+      if (angularVersion == '1') {
```

Comment:
This change is to satisfy TypeScript, loadMocks must have a string value. Also, since angularVersion is a string, it must then be compared to a string value. After the files are transpiled, the value '1' and 1 should not matter.